test/ufuzz.js

```diff
@@ -728,11 +728,12 @@ function getVarName() {
 }
 
 function createVarName(maybe, dontStore) {
-    if (!maybe || rng(2) === 1) {
-        var r = rng(VAR_NAMES.length);
-        var suffixed = rng(5) > 0;
-        var name = VAR_NAMES[r] + (suffixed ? '_' + (++loops) : '');
-        if (!dontStore && suffixed) VAR_NAMES.push(name);
+    if (!maybe || rng(2)) {
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
+        if (rng(3)) {
+            name += '_' + ++loops;
```

Comment:
Let me rephrase that, it works but not well. Creating a new variable name should have nothing to do with loops. It should be a random number.